### DISCUSSION TITLE
How do we manage releases and pre-releases

### DISCUSSION THREAD

**Discussion by dylantientcheu** [nodejs/undici#3590] [General]
A few of the issues I had have been recently closed (fixed). 
I'd like to have the version with the fixed installed. Should I install the `@next` version? It also hadn't been deployed in a while.

I'm unable to find information on the release process and frequency. What should i do?

**Comment by metcoder95** [comment]
You can track the releases through: https://github.com/nodejs/undici/releases

We are preparing releases for v7 and some backports to v6.

AFAIA, there's no fixed release schedule.
